client/js/lounge.js

```diff
@@ -1534,17 +1535,6 @@ $(function() {
 		$("#nick-value").text(nick);
 	}
 
-	function move(array, old_index, new_index) {
-		if (new_index >= array.length) {
-			var k = new_index - array.length;
-			while ((k--) + 1) {
-				this.push(undefined);
-			}
-		}
-		array.splice(new_index, 0, array.splice(old_index, 1)[0]);
-		return array;
-	}
-
```

Comment:
That code was replaced, look at this diff line 303. `typeof` because bad Shout code.